packages/agoric-cli/src/commands/perf.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable @jessie.js/no-nested-await */
+// @ts-check
+/* eslint-disable func-names */
+import {
+  iterateLatest,
+  makeCastingSpec,
+  makeFollower,
+  makeLeaderFromRpcAddresses,
+} from '@agoric/casting';
+import { execSync } from 'child_process';
+import { Command } from 'commander';
+import fs from 'fs';
+import { exit } from 'process';
+import { makeLeaderOptions } from '../lib/casting.js';
+import { normalizeAddress } from '../lib/keys.js';
+import { networkConfig } from '../lib/rpc.js';
+
+/**
+ *
+ * @param {import('anylogger').Logger} logger
+ */
+export const makePerfCommand = async logger => {
+  const perf = new Command('perf').description('Performance testing commands');
+
+  perf
+    .command('satisfaction')
+    .requiredOption(
+      '--executeOffer <filename>',
+      'filename of prepared executeOffer message',
+    )
+    .description('filename of prepared offer')
+    .requiredOption(
+      '--from [address]',
+      'address literal or name',
+      normalizeAddress,
+    )
+    .option(
+      '--keyring-backend [string]',
+      'Select keyring’s backend (os|file|kwallet|pass|test|memory) (default "os")',
+      'os',
+    )
+    .option('--home [string]', 'directory for config and data')
+    .action(async function () {
+      const opts = this.opts();
+      logger.warn({ opts });
+      const payloadStr = await fs.readFileSync(opts.executeOffer).toString();
+      const { offer } = JSON.parse(JSON.parse(payloadStr).body);
+      const { id: offerId } = offer;
+
+      const { chainName, rpcAddrs } = networkConfig;
+
+      const spec = `:published.wallet.${opts.from}`;
+
+      const leaderOptions = makeLeaderOptions({
+        sleep: 3,
```

Comment:
I wonder what sort of performance measurement we're after here. We're limited to a resolution of about 3 seconds. Given that we only observe blocks, I suppose that's as good as it gets.

Reply:
That was my thought as well. Though with blocks being every 6 seconds, 3s still adds noise.
```
0 sleep
1 sleep
2 sleep
3 poll: not available
3.1 available
4 sleep
5 sleep
6 check: available
```

So up to 2.9s off. Since high load is _the point_ of the test, I'll drop this to 0.1 and make it a declared const @arirubinstein can tweak.